test/interpreter/function-mround.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {HyperFormula} from '../../src'
+import {ErrorType} from '../../src/Cell'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function MROUND', () => {
```

Comment:
not sure how we handle -- probably random as well. Trick is, it depends on how the (imprecise) floating point operations land against the threshold. This might be dependent on js math library implementation and might be not consistent even between browsers.

Reply:
(we round exactly the same as Excel -- at least on my machine, with my js version. That might suggest that deep down we use the same low-level floating point operations. But still, I wouldn't count on consistency with this.)

Reply:
Oh, that's tough. I thought that JS is based on IE 754 and we have our own precision control so with the same settings as Excel (our defaults) we should get the same result.

Maybe we can enable the test and check if it will pass on CI? If any other developer will report it, we will disable it then